osgtest/tests/test_23_gratia.py

```diff
@@ -164,3 +166,12 @@ def test_09_config_user_vo_map(self):
                         conFileContents,
                         owner='root')
 
+    def test_10_fix_tomcat_template(self):
+        # Fix EL7 bug in Gratia template
+        if core.el_release() == 7:
+            core.skip_ok_unless_installed(tomcat.pkgname(), 'gratia-service')
+            core.config['gratia.broken_template'] = '/usr/share/gratia/server.xml.template'
+            bad_line = r'\s+sSLImplementation=.*'
+            fixed_line = ' '*15 + 'sslImplementationName="org.glite.security.trustmanager.tomcat.TMSSLImplementation"'
+            files.replace_regexpr(core.config['gratia.broken_template'], bad_line, fixed_line, owner='gratia')
+
```

Comment:
Nope because it's very much a 'hacky thing' we do to get our Frankenstein setup to work. I can make a JIRA ticket and assign it to Carl, though